bpf/Makefile.bpf

```diff
@@ -35,12 +35,16 @@ LLC   ?= llc
 HOST_CC    ?= gcc
 HOST_STRIP ?= strip
 
-ifeq ($(GOARCH),arm64)
-  HOST_CC = aarch64-linux-gnu-gcc
-  HOST_STRIP = aarch64-linux-gnu-strip
-else ifeq ($(GOARCH),amd64)
-  HOST_CC = x86_64-linux-gnu-gcc
-  HOST_STRIP = x86_64-linux-gnu-strip
+ifdef CROSS_ARCH
+  ifneq ($(CROSS_ARCH),$(GOARCH))
+    ifeq ($(GOARCH),arm64)
+      HOST_CC = aarch64-linux-gnu-gcc
+      HOST_STRIP = aarch64-linux-gnu-strip
+    else ifeq ($(GOARCH),amd64)
+      HOST_CC = x86_64-linux-gnu-gcc
+      HOST_STRIP = x86_64-linux-gnu-strip
+    endif
+  endif
 endif
```

Comment:
Nit: This is a bit too complex, as `CROSS_ARCH` is empty when not cross-compiling. This should work (see how `CROSS_ARCH` is used in `Makefile.defs`:
```
HOST_CC    ?= gcc
HOST_STRIP ?= strip

ifeq ($(CROSS_ARCH),arm64)
  HOST_CC = aarch64-linux-gnu-gcc
  HOST_STRIP = aarch64-linux-gnu-strip
else ifeq ($(CROSS_ARCH),amd64)
  HOST_CC = x86_64-linux-gnu-gcc
  HOST_STRIP = x86_64-linux-gnu-strip
endif
```

Reply:
That works as well, thanks!